public/angular/sass/app.scss

```diff
@@ -43,6 +56,14 @@ a {
   margin-right: 10px;
 }
 
+.margin-left {
+  margin-left: 10px;
+}
+
+.margin-bottom-lg {
+  margin-bottom: 20px;
+}
+
```

Comment:
@hooverlunch I think there are two approaches to this. I personally prefer the "un-semantic" approach: keep the classes very generic and combine them to describe different elements. This allows for better class reuse and keeps things simple. I've noticed that Aggie takes different approach so I will refactor.